NodeJS/nestjs/src/app.service.ts

```diff
@@ -0,0 +1,19 @@
+import { Injectable } from '@nestjs/common';
```

Comment:
One of the challenges is try to keep everyting in a single file, so could you try moving this and the app.module.ts to the main.ts file please?